src/components/place/CommentField.tsx

```diff
@@ -14,19 +14,21 @@ interface Props {
 
 export const CommentField = ({ user, review, placeId, style }: Props) => {
   return (
-    <CommentContainer style={style}>
-      <UserImage url={user.imageUrl} active={Boolean(review)} />
-      <div style={{ height: "2em", display: "flex", alignItems: "center" }}>
-        <SpeakTriangle />
-      </div>
-      <RatingContainer>
-        {review ? (
-          <CommentOnly review={review} />
-        ) : (
-          <NewCommentForm placeId={placeId} />
-        )}
-      </RatingContainer>
-    </CommentContainer>
+    user && (
```

Comment:
Early `return null` instead?

Or handle no user by rendering a default user icon and something like “Logga in för att kommentera”

Reply:
I figured rendering "logga in för att kommentera" on every card would look dumb, but we may need some UX expertise here.